riscv/src/register/mie.rs

```diff
@@ -100,7 +100,7 @@ pub fn clear_interrupt<I: CoreInterruptNumber>(interrupt: I) {
 /// Enabling interrupts might break critical sections or other synchronization mechanisms.
 /// Ensure that this is called in a safe context where interrupts can be enabled.
 #[inline]
-pub unsafe fn set_interrupt<I: CoreInterruptNumber>(interrupt: I) {
+pub unsafe fn enable_interrupt<I: CoreInterruptNumber>(interrupt: I) {
```

Comment:
Is there a reason you're not including the `is_enabled`, `enable`, `disable` methods for the `Mie` type, but are including them for the `Sie` type?

Reply:
Sorry, the changes just didn't show up in the diff I was looking at, never mind. :sweat_smile: